cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -588,14 +588,13 @@ std::unique_ptr<column> build_output_column(size_type num_rows,
     return weights[i] == 0;
   };
   // whether or not this particular tdigest is a stub
-  auto is_stub_digest = cuda::proclaim_return_type<size_type>(
-    [offsets = offsets->view().begin<size_type>(), is_stub_weight] __device__(size_type i) {
-      return is_stub_weight(offsets[i]) ? 1 : 0;
-    });
+  auto is_stub_digest = [offsets = offsets->view().begin<size_type>(), is_stub_weight] __device__(
```

Comment:
This lambda is going to be used in line 643, inside another lambda which is enclosed in `proclaim_return_type`.

Reply:
I would love a comment for that so the reader does not have to puzzle the pieces together